static/js/composebox_typeahead.js

```diff
@@ -753,6 +767,8 @@ exports.content_highlighter = function (item) {
         return typeahead_helper.render_typeahead_item({ primary: item });
     } else if (this.completing === 'topic_list') {
         return typeahead_helper.render_typeahead_item({ primary: item });
+    } else if (this.completing === 'time_jump') {
+        return typeahead_helper.render_typeahead_item({ primary: item });
```

Comment:
The typeahead seems to handle pasting a completed `!time` syntax badly:

![image](https://user-images.githubusercontent.com/2746074/82499338-5e908680-9aa6-11ea-937f-e2d5936d325c.png)

Reply:
Fixed with a regex change